jobs/jobs.go

```diff
@@ -4,71 +4,111 @@
 package jobs
 
 import (
-	"sync"
+	"context"
+	"time"
 
 	l4g "github.com/alecthomas/log4go"
-	ejobs "github.com/mattermost/platform/einterfaces/jobs"
 	"github.com/mattermost/platform/model"
 	"github.com/mattermost/platform/store"
-	"github.com/mattermost/platform/utils"
 )
 
-type Jobs struct {
-	startOnce sync.Once
+func CreateJob(store store.Store, jobType string, jobData map[string]interface{}) (*model.Job, *model.AppError) {
```

Comment:
Minor nit, Is there a reason why we're passing in store store.Store vs how we do it other places?  0/5 feels like a different pattern.